internal/ingress/controller/nginx.go

```diff
@@ -799,6 +828,31 @@ func configureDynamically(pcfg *ingress.Configuration, port int, isDynamicCertif
 	return nil
 }
 
+func updateStreamConfiguration(streams []ingress.Backend) error {
+	conn, err := net.Dial("unix", nginxStreamSocket)
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
+
+	buf, err := json.Marshal(streams)
+	if err != nil {
+		return err
+	}
+
+	_, err = conn.Write(buf)
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintf(conn, "\r\n")
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
```

Comment:
you already have this above, why do you need it again?

Reply:
fixed